server/project-service/src/main/java/edu/illinois/ncsa/incore/service/project/controllers/ProjectController.java

```diff
@@ -404,6 +437,7 @@ public List<DatasetResource> listDatasetsOfProject(
         @Parameter(name = "Skip the first n results", description = "Number of results to skip.") @QueryParam("skip") @DefaultValue("0") int offset,
         @Parameter(name = "Limit the number of results", description = "Maximum number of results to return.") @QueryParam("limit") @DefaultValue("100") int limit,
         @Parameter(name = "Filter by type", description = "Filter datasets by type") @QueryParam("type") String type,
+        @Parameter(name = "Text to search ") @QueryParam("searchText") String searchText,
```

Comment:
Just nitpick,  can we just directly use Search Text instead of Text to search? Other methods are using Search Text

Reply:
Updated.